test/pricer_rule/utils.js

```diff
@@ -60,24 +62,30 @@ module.exports.createOrganization = async (accountProvider) => {
  * Creates and returns the tuple:
  *      (tokenRules, organizationAddress, token)
  */
-module.exports.createTokenEconomy = async (accountProvider) => {
+module.exports.createTokenEconomy = async (accountProvider, config = {}, eip20TokenConfig = {}) => {
   const {
     organization,
     organizationOwner,
     organizationWorker,
   } = await this.createOrganization(accountProvider);
 
-  const token = await this.createEIP20Token();
+  const eip20TokenDecimals = eip20TokenConfig.decimals || 18;
+  const createEIP20TokenConfig = {
+    symbol: eip20TokenConfig.symbol || 'OST',
+    name: eip20TokenConfig.name || 'Open Simple Token',
+    decimals: eip20TokenDecimals,
+  };
+  const token = await this.createEIP20Token(createEIP20TokenConfig);
 
   const tokenRules = await TokenRulesSpy.new();
 
-  const baseCurrencyCode = 'OST';
-
-  const conversionRate = 315;
+  const baseCurrencyCode = config.baseCurrencyCode || 'OST';
 
-  const conversionRateDecimals = 2;
+  // (For 1 OST = 1 BT)
```

Comment:
What is this comment for?

Reply:
To derive 1 OST = 1 BT, if conversionRateDecimals = 5, then conversionRate needs to be 10^5.
I will add more verbosity in the comment.

Reply:
I am confused, however, in my understanding:

In case of 3.15, the conversionRate is 315 and conversionRateDecimals is 2.

I am not sure what do you mean by saying conversionRate is 10 ^ 5?

Reply:
**Requirement**:

Derive 1 OST = 1 BT 
conversionRateDecimalsFromBaseCurrencyToToken = 5
```
_payCurrencyAmount
            .mul(
                conversionRateFromBaseCurrencyToToken
            )
            .mul(
                10 ** uint256(tokenDecimals)
            )
            .div(
                10 ** conversionRateDecimalsFromBaseCurrencyToToken
            )
            .div(
                _baseCurrencyPriceInPayCurrency
            );
```
What should be conversionRateFromBaseCurrencyToToken value in above method so that (conversionRateFromBaseCurrencyToToken) and (10 ** conversionRateDecimalsFromBaseCurrencyToToken) nullify each other.

if we pass conversionRateFromBaseCurrencyToToken = 10^5 than in the calculation, they nullify each other.

Please also see below gist:
https://gist.github.com/abhayks1/ca49a7db6c0e89ddf4dd9ba568abfac1

```
In case of 3.15, the conversionRate is 315 and conversionRateDecimals is 2.
```
This case is different because OST/BT conversionRate is 315 and not 1:1.

Reply:
```
In case of 3.15, the conversionRate is 315 and conversionRateDecimals is 2.
```
This case is different because OST/BT conversionRate is 315 and not 1:1. 
Here 1 OST = 3.15 BT and not (1 OST = 1 BT)

Reply:
```
1 OST = (CR/10^CRD) BT
For 1 OST = 1 BT
CR/10^CRD = 1
Which means 
CR = 10^CRD
```
So If CD = 5 then CRD = 10 ^5
If CD = 2 then CR = 10^2